src/main/menu.ts

```diff
@@ -250,3 +250,131 @@ export const setMainMenu = ({ mainWindow, openRecentRev = [] }: MainMenuArgs) =>
     const menu = Menu.buildFromTemplate(template);
     Menu.setApplicationMenu(menu);
 };
+
+export const setEmptyMenu = () => {
```

Comment:
I hate it....

You didn't disable the menu, you made a new menu that is disabled. Now we have to maintain 2 menus. If we change the main menu, we now also have to remember to update the second disabled menu. That will be fun...

Instead, could you please add a `enabled: boolean = true` property to the arguments object? Then you'll only have to add the property `enabled` for all menu items you want to disable.

Reply:
Oh yeah, duh. For a while I was trying to dynamically get the current menu and map the items to make them all disabled, but that didn't work at all, so I just thought I needed to make a new menu. Didn't think to just add the enabled stuff to the original lol